src/plone/api/content.py

```diff
@@ -523,27 +525,28 @@ def get_view(name=None, context=None, request=None):
     # available, because the __init__ of said view will contain
     # errors in client code.
 
-    # Get all available views...
-    sm = getSiteManager()
-    available_views = sm.adapters.lookupAll(
-        required=(providedBy(context), providedBy(request)),
-        provided=Interface,
-    )
-    # and get their names.
-    available_view_names = [view[0] for view in available_views]
-
-    # Raise an error if the requested view is not available.
-    if name not in available_view_names:
-        raise InvalidParameterError(
-            "Cannot find a view with name '{name}'.\n"
-            "Available views are:\n"
-            "{views}".format(
-                name=name,
-                views="\n".join(sorted(available_view_names)),
-            ),
+    try:
+        return getMultiAdapter((context, request), name=name)
+    except ComponentLookupError: 
+        #Getting all available views
+        sm = getSiteManager()
+        available_views = sm.adapters.lookupAll(
+            required=(providedBy(context), providedBy(request)),
+            provided=Interface,
         )
-    return getMultiAdapter((context, request), name=name)
-
+        
+        # Check if the requested view is available by getting the names of all available views
```

Comment:
We tend to follow the black line length limit of 88 characters:
```suggestion
        # Check if the requested view is available
        # by getting the names of all available views
```
Usually we have `pre-commit` to check this, but it is not yet in this package.
So do this at will